src/envoy/utils/utils.cc

```diff
@@ -70,6 +70,20 @@ bool GetIpPort(const Network::Address::Ip* ip, std::string* str_ip, int* port) {
   return false;
 }
 
+bool GetDestinationUID(const envoy::api::v2::core::Metadata& metadata,
+                       std::string* uid) {
+  const auto filter_it = metadata.filter_metadata().find("mixer");
```

Comment:
can we use the same constant string in the mixer filter factory and here?

Reply:
There's no shared filter factory between http/tcp, while this applies to both. I defined a new constant for this purpose.